panel-toplevel.c:2681: undefined reference to `g_clear_signal_handler

#### Expected behaviour
Building for Rocky Linux 8.5 (glibc 2.56.4) shall be done without an error

#### Actual behaviour
Building stops with an error:
mate_panel-panel-toplevel.o: In function `panel_toplevel_disconnect_attached':
/builddir/build/BUILD/mate-panel-c4602959df4f2856e49ebf6b6c7a03be00d311f4/mate-panel/panel-toplevel.c:2681: undefined reference to `g_clear_signal_handler'
collect2: error: ld returned 1 exit status
make[4]: *** [Makefile:877: mate-panel] Error 1

#### Steps to reproduce the behaviour
Build for a system with glibc < 2.62

#### MATE general version
1.27.0

#### Package version
1.27.0

#### Linux Distribution
Rocky Linux 8.5

#### Link to bugreport of your Distribution (requirement)
Package is not available in distribution, so nowhere to report...



I believe there is a typo in commit https://github.com/mate-desktop/mate-panel/commit/293948a, where the line(s) with the unavailable function has been left there by mistake when copied and edited the code from 2.62 section above (I see it is replaced with the right code just below).

A patch (to just remove the offending lines):
```
diff -up mate-panel/mate-panel/panel-toplevel.c~ mate-panel/mate-panel/panel-toplevel.c
--- mate-panel/mate-panel/panel-toplevel.c~	2022-03-09 10:07:16.935768822 +0100
+++ mate-panel/mate-panel/panel-toplevel.c	2022-03-09 10:07:17.011769308 +0100
@@ -2678,8 +2678,6 @@ panel_toplevel_disconnect_attached (Pane
 	}
 
 	for (i = 0; i < N_ATTACH_WIDGET_SIGNALS; i++) {
-		g_clear_signal_handler (&toplevel->priv->attach_widget_signals [i],
-		                        toplevel->priv->attach_widget);
 		if (toplevel->priv->attach_widget_signals [i] != 0) {
 			g_signal_handler_disconnect (toplevel->priv->attach_widget,
 			                             toplevel->priv->attach_widget_signals [i]);

```
fixes the issue for me (build went just fine after applying it, and the panel seems to work as expected).

--- a/mate-panel/panel-toplevel.c
+++ b/mate-panel/panel-toplevel.c
@@ -2678,8 +2678,6 @@ panel_toplevel_disconnect_attached (PanelToplevel *toplevel)
 	}
 
 	for (i = 0; i < N_ATTACH_WIDGET_SIGNALS; i++) {
-		g_clear_signal_handler (&toplevel->priv->attach_widget_signals [i],
-		                        toplevel->priv->attach_widget);
 		if (toplevel->priv->attach_widget_signals [i] != 0) {
 			g_signal_handler_disconnect (toplevel->priv->attach_widget,
 			                             toplevel->priv->attach_widget_signals [i]);

